mechanics/mechanics.py

```diff
@@ -0,0 +1,37 @@
+from random import choice, uniform
+
+from model.Unit import Unit
+
+golden = (1 + 5 ** 0.5) / 2
+
+
+def generate_unit(round: int) -> Unit:
+    # profession = choice(['magician', 'dps', 'tank', 'sniper'])
```

Comment:
commented code.

Reply:
leftover comments

Reply:
profession choice will be implemented very soon